### PR TITLE
naml 1.0.3 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/naml.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    (testpath/"service.yaml").write <<~EOS
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: brewtest
+    EOS
+
+    assert_match "Application autogenerated from NAML v#{version}",
+      shell_output("cat #{testpath}/service.yaml | #{bin}/naml codify")
```
Comment (carlocab) [inline Formula/naml.rb:31]:
```suggestion
      pipe_output("#{bin}/naml codify", (testpath/"service.yaml").read)
```

File: Formula/naml.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+class Naml < Formula
+  desc "Convert Kubernetes YAML to Golang"
+  homepage "https://github.com/kris-nova/naml"
+  url "https://github.com/kris-nova/naml/archive/refs/tags/v1.0.3.tar.gz"
+  sha256 "0842633268b06be82db4dd10c3c938f756f613c44c15c2d935b933409da8c4bd"
+  license "Apache-2.0"
+  head "https://github.com/kris-nova/naml.git", branch: "main"
+
+  depends_on "go" => [:build, :test]
```
Comment (carlocab) [inline Formula/naml.rb:9]: Is Go needed at runtime?

Comment (chenrui333) [reply Formula/naml.rb:9]: Yeah, but it does not need the latest go though.

Comment (chenrui333) [reply Formula/naml.rb:9]: Actually, it is a good idea to have it as runtime dependency. Good call.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**